custom_components/tuya_local/devices/nexsmart_air_2.yaml

```diff
@@ -62,13 +62,6 @@ entities:
       - id: 149
         type: boolean
         name: sensor
-        mapping:
-          - dps_val: true
-            value: "on"
-            value_on: Awake
-          - dps_val: false
-            value: "off"
-            value_off: Sleeping
 
   - entity: number
     name: Low Battery Alarm Threshold

```

Comment:
HA naming convention says only the first word in an entity name should be capitalized.